Makefile

```diff
@@ -33,7 +34,16 @@ UNAME := $(shell uname)
 
 .SILENT:
 
-all: libs
+ifndef COMPILE_FOR_FUZZING
+ALL_TARGET := examples
+else
+ALL_TARGET := fuzzers
+endif
+
+all: $(ALL_TARGET)

```

Comment:
I don't think this is needed. We can keep it the way it was since you added `fuzzing:` below anyway

Reply:
I used this becasue with the fuzzing build flags the examples do not build correctly. I will leave it as it was and add a warning message on the `configure-fuzzing.sh` script to tell the user to run only `make fuzzers` and not `make all` or `make`.